docs/container-set-template.md

```diff
@@ -116,3 +116,52 @@ Example B: Lopsided requests, e.g. `a -> b` where `a` is cheap and `b` is expens
 Can you see the problem here? `a` only has small requests, but the container set will use the  total of all requests. So it's as if you're using all that GPU for 10h. This will be expensive.
 
 Solution: do not use container set when you have lopsided requests.
+
+## `retryStrategy` usage
+
+> v3.3 and after
+
+You can set a `retryStrategy` to apply to all containers of a container set.
+
+This currently works differently from [retries](retries.md) for other template types.
+You can only set the `duration` between each retry and the total number of `retries`.
+
+See an example below:
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  name: containerset-with-retrystrategy
+  annotations:
+    workflows.argoproj.io/description: |
+      This workflow template is used to create a workflow with containerset with retrystrategy.
```

Comment:
```suggestion
      This workflow creates a container set with a retryStrategy.
```

- This isn't a `WorkflowTemplate`
- Also should use the same spelling as elsewhere in this doc for consistency:
  - "containerset" -> "container set"
  - "retrystrategy" -> "retryStrategy"

Although I'm not sure this annotation is needed at all, since this isn't a standalone example manifest, it's in the context of docs that already explain this.

Reply:
I see you just copied your testdata manifest here, but they shouldn't necessarily be the same, as the purposes are different. One is for explanation / demonstration, the other is for testing.